docs/project-structure.md

```diff
@@ -20,6 +20,7 @@ Sample applications are not covered.
 - [JSI.Desktop.UnitTests\JSI.Desktop.UnitTests](#JSI.Desktop.UnitTests)
 - **[Microsoft.ReactNative\Microsoft.ReactNative.vcxproj](#Microsoft.ReactNative)**
 - [Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems](#Microsoft.ReactNative.Cxx)
+- [Microsoft.ReactNative.Shared\Microsoft.ReactNative.Shared.csproj](#Microsoft.ReactNative.Managed)
```

Comment:
>Microsoft.ReactNative.Shared.csproj [](start = 32, length = 35)

I think this link must use term 'Managed instead of 'Shared'

Reply:
oops

---
In reply to: [420380580](https://github.com/microsoft/react-native-windows/pull/4793#discussion_r420380580) [](ancestors = 420380580)